### PR TITLE
guides: add trailing commas to last items in js objects/arrays

### PR DESCRIPTION
**Pull request by ejez**
guides: correct diffs to account for adding a comma before the last item when appending items to an object

### CODE REVIEW

File: src/content/guides/asset-management.md [hunk 1]
```diff
@@ -72,7 +72,8 @@ __webpack.config.js__
     output: {
       filename: 'bundle.js',
       path: path.resolve(__dirname, 'dist')
-    },
+-    }
++    },
```
Comment (EugeneHlushko) [inline src/content/guides/asset-management.md]:
```suggestion
+   },
```

**Comment by ejez** [comment]
@EugeneHlushko , don't merge yet, the issue is repeated many times in the docs, let's create one PR for all

**Comment by ejez** [comment]
Checked the guides directory, and using a regex I added a comma to the last item in an object when a new item is appended to the end of that object.
@EugeneHlushko

**Comment by montogeek** [comment]
Me too, this problem is so common that there is a JS proposal for it :D

**Comment by ejez** [comment]
Another related suggestion:

the diff signs (+/-) make it difficult for the reader to copy/paste, if the diff is going to be kept, it would be better to remove the diff signs, I think the colors (green/red) would be enough for the reader.

something like this should work:
```js
function removeDiffSigns() {
  const diffElements = document.querySelectorAll(
    "code.language-diff > span.inserted, code.language-diff > span.deleted"
  );

  diffElements.forEach(function(diffElement) {
    diffElement.innerHTML = diffElement.innerHTML.replace(/^[+-]/, '');
  });
}
```

**Comment by ejez** [comment]
> This makes me want to have trailing comma rule in our configs

Agree, here is a quote from Mozilla: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

> Trailing commas (sometimes called "final commas") can be useful when adding new elements, parameters, or properties to JavaScript code. If you want to add a new property, you can simply add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome.
> 
> JavaScript has allowed trailing commas in array literals since the beginning, and later added them to object literals (ECMAScript 5) and most recently (ECMAScript 2017) to function parameters.

**Comment by EugeneHlushko** [comment]
@ejez do you want to re-work this PR into changing all "diff" code snippets in guides to have trailing comma and reduce the amount of diff size on them?

**Comment by ejez** [comment]
Will do when I get home... 

You mean all code snippets, not only those having diffs.

**Comment by ejez** [comment]
@EugeneHlushko 
done

**Pull request merged**